lib/create-organization-api.js

```diff
@@ -116,6 +120,42 @@ export default function createOrganizationApi ({ http }) {
       .then((response) => wrapSpaceMembershipCollection(http, response.data), errorHandler)
   }
 
+  /**
+   * Gets an Team Membership
+   * @memberof ContentfulOrganizationAPI
+   * @param {string} id - Team Membership ID
+   * @return {Promise<TeamMembership>} Promise for an Team Membership
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * client.getOrganizations()
+   * .then((organization) => organization.getTeamMembershipByTeam('teamId', 'teamMembership_id'))
+   * .then((teamMembership) => console.log(teamMembership))
+   * .catch(console.error)
+   */
+  function getTeamMembershipByTeam (teamId, teamMembershipId) {
```

Comment:
I don't like the name `getTeamMembershipByTeam`. Would there ever be another way to get a team membership? I think prob not.

let's drop the `byTeam` part from the methods

Reply:
Well we also have another endpoint to be added soon which is get all `/organization/orgId/team_memberships` and for that we do not want the update and delete which is why I have two wrapper methods;
However maybe I can make these wrapTeamMemberships and for the one case of all org team memberships I will name it like that: `wrapOrganizationTeamMemberships` WDYT?

Reply:
this whole api is very confusing to me.

I think your proposal is good: organizationTeamMemberships and teamMemberships. They're separate entities, one has update/delete methods and one returns an "organizationTeamMembership" which is readOnly.

Maybe also elaborate in the jsdoc strings a little more about this api, because for me as a reviewer, it's super confusing. (Also spacemembers, spacememberships, etc)

Reply:
I see, makes sense, I will separate the organizationTeamMembership.
Ill add more documentation too. Thank you.